explanation/performance.rst

```diff
@@ -39,6 +39,7 @@ following sections.
 * Obtain the hierarchical map of key computing elements using :ref:`hwloc and lstopo <perf-tune-hwloc>`
 * Check and control CPU governors, power and frequency with :ref:`cpupower <perf-tune-cpupower>`
 * Dynamic, adaptive system tuning :ref:`with TuneD <perf-tune-tuned>`
+* Energy Performance Preference :ref:`EPP <perf-epp>`
```

Comment:
You've added the link here, but you also need to add the page itself to the toctree at the bottom of the page - it's not currently showing up in the navigation

Reply:
Ah I see, updated